src/features/user/MFV2/ContributionsMap/Markers/SinglePointMarkers.tsx

```diff
@@ -0,0 +1,108 @@
+import { Marker } from 'react-map-gl-v7';
+import { useMemo } from 'react';
+import RegisteredTreeIcon from '../../../../../../public/assets/images/icons/myForestV2Icons/RegisteredTreeIcon';
+import NaturalRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/NaturalRegeneration';
+import Mangroves from '../../../../../../public/assets/images/icons/myForestV2Icons/Mangroves';
+import ManagedRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/ManagedRegeneration';
+import Agroforestry from '../../../../../../public/assets/images/icons/myForestV2Icons/Agroforestry';
+import UrbanRestoration from '../../../../../../public/assets/images/icons/myForestV2Icons/UrbanRestoration';
+import Conservation from '../../../../../../public/assets/images/icons/myForestV2Icons/Conservation';
+import TreePlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/TreePlanting';
+import OtherPlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/OtherPlanting';
+import themeProperties from '../../../../../theme/themeProperties';
+import { useMyForestV2 } from '../../../../common/Layout/MyForestContextV2';
+
+type Classification =
+  | 'natural-regeneration'
+  | 'mangroves'
+  | 'managed-regeneration'
+  | 'agroforestry'
+  | 'urban-planting'
+  | 'conservation'
+  | 'large-scale-planting'
+  | 'other-planting';
+interface ProjectTypeIconProps {
+  purpose: string;
+  classification: Classification;
+}
+
+const ProjectTypeIcon = ({ purpose, classification }: ProjectTypeIconProps) => {
+  const getMarkerColor = (purpose: string) => {
+    switch (purpose) {
+      case 'conservation':
+        return `${themeProperties.mediumBlue}`;
+      case 'restoration':
+        return `${themeProperties.electricPurple}`;
+      default:
+        return `${themeProperties.primaryDarkColorX}`;
+    }
```

Comment:
The logic doesn't seem right.

As I understand there are different marker colors for projects with:

1. purpose = "trees" and unitType = "tree"
2. purpose = "trees" and unitType = "m2"
3. purpose = "conservation" (and unitType = "m2" which is always the case)